{{cookiecutter.github_repository}}/provisioner/roles/celery/defaults/main.yml

```diff
@@ -1,2 +1,9 @@
----
+{%raw%}---
+celery_user: www-data
+celery_group: www-data
 celery_concurrency: 4
+celery_log_file: "/var/log/celery.log"
```

Comment:
@jainmickey Can we have logrotate complaint log dir for this too? You can check out the `project_data` role for our log for uwsgi process.